runtime/near-vm-runner-standalone/contract/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "basic-contract"
```

Comment:
Prefer keeping the naming of the crate the same as the folder name.

Reply:
Done.